src/main/java/md/forum/forum/models/Like.java

```diff
@@ -21,15 +21,14 @@ public class Like {
     private boolean upvote;
     @Column(name = "create_date")
     private Date createDate;
-    @JoinColumn(name = "post_id",foreignKey = @ForeignKey(name = "fk_like_post"))
+    @JoinColumn(name = "post_id", foreignKey = @ForeignKey(name = "fk_like_post"))
```

Comment:
Please, do not make additional refactor if it is not related to your ticket. We can raise separate ticket for refactor.